wso2is_analytics/hieradata/dev/wso2/wso2is_analytics/pattern-1/default.yaml

```diff
@@ -0,0 +1,417 @@
+#    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+#
+#  wso2_gov_db:
+#    name: WSO2_GOV_DB
+#    description: The datasource used for gov registry
+#    driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
+#    url: jdbc:mysql://192.168.100.1:3306/WSO2_REG_DB?autoReconnect=true
+#    username: "%{hiera('wso2::datasources::mysql::username')}"
+#    password: "%{hiera('wso2::datasources::mysql::password')}"
+#    jndi_config: jdbc/WSO2_GOV_DB
+#    max_active: "%{hiera('wso2::datasources::common::max_active')}"
+#    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+#    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+#    validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
+#    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+#    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+#
+#  wso2_user_db:
+#    name: WSO2_USER_DB
+#    description: The datasource is used for user mangement and userstore
+#    driver_class_name: com.mysql.jdbc.Driver
+#    url: jdbc:mysql://192.168.100.1:3306/WSO2_USER_DB?autoReconnect=true
+#    username: "%{hiera('wso2::datasources::mysql::username')}"
+#    password: "%{hiera('wso2::datasources::mysql::password')}"
+#    jndi_config: jdbc/WSO2_USER_DB
+#    max_active: "%{hiera('wso2::datasources::common::max_active')}"
+#    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+#    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+#    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+#    validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
+#    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+
+wso2::analytics_datasources:
+  wso2_analytics_event_store_db:
+    name: WSO2_ANALYTICS_EVENT_STORE_DB
+    description: The datasource used for analytics record store
+    driver_class_name: "%{hiera('wso2::datasources::h2::driver_class_name')}"
+    url: jdbc:h2:repository/database/ANALYTICS_EVENT_STORE;AUTO_SERVER=TRUE;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000
+    username: "%{hiera('wso2::datasources::common::username')}"
+    password: "%{hiera('wso2::datasources::common::password')}"
+    max_active: "%{hiera('wso2::datasources::common::max_active')}"
+    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+    validation_query: "%{hiera('wso2::datasources::h2::validation_query')}"
+    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+    default_auto_commit: false
+    initial_size: 0
+    test_while_idle: true
+    min_evictable_idle_time_millis: 4000
```

Comment:
IMO it would be better to assume `pattern-1` to have a MySQL backend for the event stores.